src/data/complex/exponential.lean

```diff
@@ -503,11 +503,22 @@ mul_div_cancel' _ two_ne_zero'
 lemma two_cosh : 2 * cosh x = exp x + exp (-x) :=
 mul_div_cancel' _ two_ne_zero'
 
+/-- The real definition of `sinh`-/
+lemma sinh_def (x : ℝ) : sinh x = (exp x - exp (-x)) / 2 :=
```

Comment:
The name `sinh_def` suggests that this is a lemma for unfolding the definition. I would rather use `sinh_eq` here.
```suggestion
lemma sinh_eq (x : ℝ) : sinh x = (exp x - exp (-x)) / 2 :=
```